sdl-core/src/main/scala/io/smartdatalake/workflow/action/spark/customlogic/CustomDsTransformer.scala

```diff
@@ -37,13 +37,14 @@ trait CustomDsTransformer[In <: Product, Out <: Product] extends Serializable {
    * @param session      Spark Session
    * @param options      Options specified in the configuration for this transformation
    * @param inputDS      Input Dataset
+   * @param dataObjectId name of the input Dataset
    * @return Transformed DataFrame
    */
-  def transform(session: SparkSession, options: Map[String, String], inputDS: Dataset[In]): Dataset[Out]
+  def transform(session: SparkSession, options: Map[String, String], inputDS: Dataset[In], dataObjectId: String): Dataset[Out]
```

Comment:
@zzeekk 
Something I forgot to ask and now notice when I use it:
The signature of transform of CustomDsTransformer is a bit inconsistent with the one expected for the Custom DSNto1Transformer. Why do we need this dataObjectId here? We don't have it in the other one